pages/common/bedops.md

```diff
@@ -0,0 +1,24 @@
+# bedops
+
+> BEDOPS: A fast, highly scalable, and easily-parallelizable genome analysis toolkit.
+> Apply efficient set operations on any number of sorted BED or Starch files such as element-of, intersection, merge, difference, partition, etc. and their logical inverses.
+
+- Take the multiset union of two or more BED or Starch files and write the sorted result to {{path/to/output_file}}:
+
+`bedops --everything {{path/to/file_1}} {{path/to/file_2}} [{{path/to/file_3}} ...] > {{path/to/output_file}}`
```

Comment:
I get what you are trying to do with the [ ...] notation. In this project, we lean more towards giving concrete examples and letting the user build up complicated examples from them. From what I see, all commands take multiple input files. So, what we can do is - keep this example to have just 2 files, and then have the next example to have 2 query files.

In this way, the user can figure out that they can add more files on command line. And we also avoid the [ ...] notation, which takes a second to figure out.

What do you think ?

Reply:
I think having some way to denote that three or more files are also possible is useful. Perhaps we could have two examples, one with two files, and another with four or five files, to make it clear that multiple files are allowed.

Reply:
Sure, I would think three files would be sufficient. We wouldn't want to make the example too long. But up to you.